go/src/koding/db/models/session.go

```diff
@@ -17,4 +24,27 @@ type Session struct {
 	SessionBegan  time.Time     `bson:"sessionBegan"`
 	LastAccess    time.Time     `bson:"lastAccess"`
 	Impersonating bool          `bson:"impersonating"`
+	Data          *Data         `bson:"data,omitempty" json:"data,omitempty"`
```

Comment:
It can be just `Data Data` without pointer and you could get rid of the nil check.